config/routes.rb

```diff
@@ -14,6 +14,7 @@
     get "" => "documents#show", as: :document
     patch "" => "documents#update"
     delete "" => "documents#destroy"
+    get "/history" => "documents#history", as: :history_path
```

Comment:
You don't want the path suffix from this - just `as: :history` or `as: :document_history` if just history is too generic.